models/Conversation.js

```diff
@@ -34,6 +34,7 @@ const ConversationSchema = new Schema({
       text: {
         type: String,
       },
```

Comment:
this needs a type right?

Reply:
```
     translations: {
        type: Object,
        English: {type: String},
        Spanish: {type: String},
        French: {type: String},
      },
```

I'm uncertain how to define a type for the dynamic keys so for now this is what I have.  
Ideally it look something like translations[language]: {type: String}